docs/docs/how-tos/nebari-aws.md

```diff
@@ -105,24 +105,42 @@ environment variables have been properly set. It is time to initialize and deplo
    mkdir nebari-aws && cd nebari-aws
    ```
 
-2. Executing the command below will generate a basic config file with an infrastructure based on **AWS**, with project name `projectname`, endpoint domain `domain`, and with the authentication mode set to **password**.
+2. Executing the command below will generate the set of questions that needs the suitable values to generate the qhub-config file with an infrastructure based on based on **AWS**, with project name `projectname`, endpoint domain `domain`, and with the authentication mode set to **password**.
```

Comment:
```suggestion
2. Executing the `nebari init --guided-init` command prompt you to respond to a set of questions, which will be used to generate the `nebari-config.yaml` file with an infrastructure based on **AWS**.
```

Reply:
@viniciusdc @iameskild  do we have the reference doc for the `config` file in the docs? might be worth x-linking here

Reply:
These docs (probably slightly outdated) try to explain each section of the config.
https://docs.qhub.dev/en/latest/source/installation/configuration.html

Reply:
Looks like we're covering `nebari-config.yaml` in #115 -- I've opened a new issue #188 to keep track of cross-references. So, closing this discussion as resolved. :)